README.md

```diff
@@ -84,50 +81,42 @@ import "github.com/ProtonMail/gopenpgp/crypto"
 
 ### Encrypt / Decrypt with password
 
-```go
-import "github.com/ProtonMail/gopenpgp/helper"
+```gotemplate
+import "github.com/ProtonMail/gopenpgp/v2/helper"
 
-const password = "my secret password"
+const password = []byte("hunter2")
 
 // Encrypt data with password
-armor, err := helper.EncryptMessageWithToken(password, "my message")
+armor, err := helper.EncryptMessageWithPassword(password, "my message")
 
 // Decrypt data with password
-message, err := helper.DecryptMessageWithToken(password, armor)
+message, err := helper.DecryptMessageWithPassword(password, armor)
 ```
 
-To use more encryption algorithms:
-```go
-import "github.com/ProtonMail/gopenpgp/constants"
-import "github.com/ProtonMail/gopenpgp/helper"
-
-// Encrypt data with password
-armor, err := helper.EncryptMessageWithTokenAlgo(password, "my message", constants.ThreeDES)
-
-// Decrypt data with password
-message, err := helper.DecryptMessageWithToken(password, armor)
-```
+To encrypt binary data or use more advanced modes:
+```gotemplate
+import "github.com/ProtonMail/gopenpgp/v2/constants"
 
-To encrypt binary data, reuse the key multiple times, or use more advanced modes:
-```go
-import "github.com/ProtonMail/gopenpgp/constants"
+const password = []byte("hunter2")
 
-var key = crypto.NewSymmetricKeyFromToken("my secret password", constants.AES256)
 var message = crypto.NewPlainMessage(data)
+// Or
+message = crypto.NewPlainMessageFromString(string)
 
 // Encrypt data with password
-encrypted, err := key.Encrypt(message)
+encrypted, err := EncryptMessageWithPassword(message, password)
+// Encrypted message in encrypted.GetBinary() or encrypted.GetArmored()
 
 // Decrypt data with password
-decrypted, err := key.Decrypt(password, encrypted)
+decrypted, err := DecryptMessageWithPassword(encrypted, password)
 
 //Original message in decrypted.GetBinary()
 ```
 
 ### Encrypt / Decrypt with PGP keys
 
-```go
-import "github.com/ProtonMail/gopenpgp/helper"
+```gotemplate
```

Comment:
Why not simply go to have syntax highlight?

Reply:
My IDE complained a lot about go not starting with `package`.
I used gotemplate that locally gave highlighting but no such errors.
Since it's not compatible with github I'll revert this change